packages/gatsby-theme-blog/src/components/post-date.js

```diff
@@ -0,0 +1,15 @@
+/** @jsx jsx */
+import { Styled, jsx } from "theme-ui"
+
+const PostDate = props => (
+  <Styled.p
+    sx={{
+      fontSize: 1,
+      mt: -3,
```

Comment:
I'm guessing this is from moving things around, but generally it's not great to use margin at the outer level of a component. I wonder if we could handle some of the margin here with an outer/wrapping component instead